files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md

```diff
@@ -17,7 +17,7 @@ Al finalizar este tutorial tendrás un juego _demo_ completamente funcional: [Cy
 
 ## Phaser framework
 
-[Phaser](http://phaser.io/) es un _framework_ para construir juegos, de móvil o escritorio, en HTML5 . Es nuevo pero está creciendo velozmente gracias a la apasionada comunidad involucrada en el proceso de desarrollo. Puedes chequearlo [en GitHub](https://github.com/photonstorm/phaser) donde se encuentra como _open source._ Lee [la documentación](http://docs.phaser.io/) en línea y recorre su gran colección de [ejemplos](http://examples.phaser.io/). El _framework_ Phaser provee una serie de herramientas que acelerarán el desarrollo y te ayudaran a manejar las tareas genéricas necesarias para completar tu juego, para que así puedas concentrarte en la idea del juego en sí.
+[Phaser](http://phaser.io/) es un _framework_ para construir juegos, de móvil o escritorio, en HTML5 . Es nuevo pero está creciendo velozmente gracias a la apasionada comunidad involucrada en el proceso de desarrollo. Puedes chequearlo [en GitHub](https://github.com/photonstorm/phaser) donde se encuentra como _open source._ Lee [la documentación](https://phaser.io/docs/2.2.2/) en línea y recorre su gran colección de [ejemplos](https://phaser.io/examples). El _framework_ Phaser provee una serie de herramientas que acelerarán el desarrollo y te ayudaran a manejar las tareas genéricas necesarias para completar tu juego, para que así puedas concentrarte en la idea del juego en sí.
```

Comment:
El enlace automáticamente dirige a la versión actual, también es el enlace utilizado en la versión en inglés, si crees que es mejor apuntar a una versión, sugerir el cambio en inglés y una vez aprobado hacer el cambio aquí

```suggestion
[Phaser](http://phaser.io/) es un _framework_ para construir juegos, de móvil o escritorio, en HTML5 . Es nuevo pero está creciendo velozmente gracias a la apasionada comunidad involucrada en el proceso de desarrollo. Puedes chequearlo [en GitHub](https://github.com/photonstorm/phaser) donde se encuentra como _open source._ Lee [la documentación](https://phaser.io/docs/) en línea y recorre su gran colección de [ejemplos](https://phaser.io/examples). El _framework_ Phaser provee una serie de herramientas que acelerarán el desarrollo y te ayudaran a manejar las tareas genéricas necesarias para completar tu juego, para que así puedas concentrarte en la idea del juego en sí.
```

Reply:
It is better to target a specific version, because the tutorial is made for Phaser 2.2.2 which is somewhat different in structure than 3. Now that I think about it it would also be good to mention that a less updated version of the same is used. So that the user does not get so confused to see that the way to use Phaser is different than the one shown in the tutorial.

Reply:
I see, so please [suggest the change](https://github.com/mdn/content/edit/main/files/en-us/games/tutorials/html5_gamedev_phaser_device_orientation/index.md) on the [`en-US`](https://developer.mozilla.org/en-US/docs/Games/Tutorials/HTML5_Gamedev_Phaser_Device_Orientation) version, once approved we can merge this.

Thank you so much !

Reply:
Hello ! @dabl03 I hope your having a great day, as this have already two weeks without action, I'm goint to merge this without the version number, once you suggest the change in the `en-US` article we can update this with that change.

Thank you so much for all your help